### DISCUSSION TITLE
multisite - déclaration de plusieurs domaines pour un site - mode pro

### DISCUSSION THREAD

**Discussion by tromeurf** [AmauriC/tarteaucitron.js#1130] [General]
Bonjour,
après avoir essayé dans l'interface avec un compte pro et n'ayant pas trouvé de discussion à ce propos ,
je bloque sur un sujet :

Est-il possible de déclarer plusieurs domaines pour un site ?

Actuellement je suis embêté car je voudrais utiliser le mode automatique sur mon multisite.
J'ai un site principal et une dizaine de sous-domaines.
Pour l'instant il me semble que je doive créer un site pour chaque domaine et sous-domaine, + mes environnements de dev et de preprod.

Y a t'il une solution ? ou une évolution envisagée pour éviter d'avoir 30 sites déclarés pour une même config ?

Merci d'avance

**Comment by tbego** [comment]
Hello !
Je suis également dans le même cas.

Si jamais quelqu'un à une astuce/solution à partager à ce sujet, ca serait top :)

Merci d'avance.

**Comment by AmauriC** [comment]
Salut 👋

Il est possible de ne déclarer qu'un site sur la console tarteaucitron et d'utiliser la même balise pour tous les sites:

```
<script src="https://tarteaucitron.io/load.js?domain=mondomaineprincipale.com&uuid=xxx"></script>
```

Pour partager le consentement sur les sous domaines, il faut dans les réglages changer la valeur du "Nom de domaine sur lequel sera posé le cookie" par: .mondomaineprincipale.com (le point au début est important).

La seule différence sera que les stats sur tarteaucitron.io seront toutes regroupées sur le même domaine.

**Reply by tbego** [reply]
Merci beaucoup pour ce retour rapide !

**Reply by tromeurf** [reply]
Yes, merci beaucoup !
